ci/bin/format.dart

```diff
@@ -507,7 +510,13 @@ class JavaFormatChecker extends FormatChecker {
     if (failures.isEmpty) {
       return true;
     }
-    return applyPatch(failures);
+    if (failures.first == _javaFormatErrorString) {
```

Comment:
Can we tighten this a bit:

failures.length == 1 && failures.first == _javaFormatErrorString

Reply:
Done, FWIW if we in the future return more strings the patch will fail to apply but that is probably a better error condition than swallowing the patch info.